### PR TITLE
[Magiclysm] Add seven more spells (it's a pattern)

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Mods "Add seven druid, earthshaper, and technomancer spells"

#### Purpose of change

I have more spell ideas and I cannot be stopped.

#### Describe the solution

Add another seven spells:

1. Shutdown (Technomancer): Stun a robot, cyborg, or golem (the original reason for the spell's existence).  Uses mod_moves rather than the stun effect, so it does work on robots.
2. Speak through the Aether (Technomancer): Summon a magical radio.  Invented before radio existed, it grew much less popular when people had radios and all of a sudden anyone could listen in on some random wizard's private conversations. 
3. Power Supply (Technomancer): A magical UPS, for the technomancer who has no bionic enhancements but still wants to power their gear.
4. Voltaic Touch (Technomancer): Magical welding through the power of electricity. I originally thought about putting this into Stormshaper but the lore document says that Stormshaper has more power behind its electricity but less control, so Technomancer it is.
5. Granite Aegis (Earthshaper): Create a wall of rock around yourself to give yourself a breather.  It mostly only works when standing on natural earth, stone, and rock (I'll probably add concrete here too) but does create a literal rock wall. 
image
(vision thanks to debug clairvoyance)
6. Summon Tanglevine (Druid): Create a vine that grabs enemies, pulls them in, and constricts them.  It doesn't do a lot of damage and can't move once summoned but it's very good at holding them in place if they get to close.
7. Spear of Brambles (Druid): Turn a stick into a spear that has a slowing sap dripping from it.  With the enhanced version, you no longer need the stick to cast it (lorewise I'm assuming you go from needing a big stick to being able to cast it from a twig or a bit of bark, which isn't worth tracking).

#### Testing

Tanglevine pulls in enemies and fails on larger enemies (in testing it successfully grabbed and crushed a manhack, thank you tanglevine), Spear of Brambles does good spear damage, Granite Aegis works on concrete and railroad beds, Voltaic Touch does a bit of electric damage (not separated in the log damage listing but unarmed damage went from ~5 to ~11 in testing, in line with the +6 electric damage bonus), and Power Supply, Shutdown, and Speak Through the Aether do what I want them too.

### CODE REVIEW

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 1]
```diff
@@ -1386,5 +1386,86 @@
...
+    "flags": [ "TRADER_AVOID", "ONLY_ONE", "NO_DROP", "NO_SALVAGE", "NO_UNLOAD", "NO_RELOAD", "IS_UPS" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 1 } } ],
+    "relic_data": { "charge_info": { "recharge_type": "periodic", "time": "2 s", "regenerate_ammo": true } },
+    "//": "Recharge time is 2 seconds due to bug #48019, making it actually 1 second. Reduce to 1 second if that ever gets fixed."
+  },
+  {
+    "id": "technomancer_welder",
+    "type": "TOOL",
+    "name": { "str": "voltaic touch" },
+    "description": "Your hands may currently spark with electricity powerful enough to heat metal to searing temperatures and weld it together.",
```
Comment (TheShadowFerret) [inline data/mods/Magiclysm/items/ethereal_items.json:1438]: No electric damage if you punch with it?

Comment (Standing-Storm) [reply data/mods/Magiclysm/items/ethereal_items.json:1438]: ...you know, I hadn't even thought of that, but you're right, _someone_ is going to try it.

**Comment by Maleclypse** [comment]
Needs a rebase

**Comment by Standing-Storm** [comment]
Should be all ready now

**Pull request merged**